pages/studentPortal/web/React/github.tsx

```diff
@@ -0,0 +1,90 @@
+import { Router, useRouter } from "next/router";
+import React from "react";
+import Card, { CardData } from "../../../../components/coding/Card";
+import LessonComponent, {
+  LessonComponentData,
+} from "../../../../components/coding/studentPortal/LessonComponent";
+import { Button } from "../../../../components/ui/Button";
+import Navbar from "../../../../components/ui/Navbar";
+
+const Github = ({ lessonComponents }) => {
+  const router = useRouter();
+
+  const handleContinue = () => {
+    router.push("/studentPortal/web/React");
+  };
+  return (
+    <>
+      <div className="grid grid-cols-1 gap-8 px-4 pt-4 m-8 sm:px-12">
+        {lessonComponents.map((it) => (
+          <LessonComponent data={it} />
+        ))}
+      </div>
+      <div className="flex my-8 mr-8 sm:justify-end">
+        <Button onClick={handleContinue} label="Continue" />
+      </div>
+    </>
+  );
+};
+
+export async function getServerSideProps({ params }) {
+  const lessonComponents: LessonComponentData[] = [
+    {
+      component: "title",
```

Comment:
Nice work! This is good for now, but we need to move this content into Sanity.io, if you're interested in helping set that up, let me know!

Reply:
yeah.  I definitely would like to do that!

Reply:
I tried to set up the webhook and got the url, etc. but ran into this error when trying to make a submission via axios.  
`Access to XMLHttpRequest at 'https://hooks.slack.com/services/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' from origin 'http://localhost:3001' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: It does not have HTTP ok status.`
Any idea on how to bypass/fix that when using the local server to test? 

i did test out the webhook using a curl request and was able to get a message sent to the channel.